### PR TITLE
Updates command names and flags

### PR DESCRIPTION
**Pull request by SakuraBlossom**

### CODE REVIEW

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 1]
```diff
@@ -87,22 +88,21 @@ public synchronized CommandResult execute(String commandText)
 
     @Override
     public synchronized void eagerEvaluate(String commandText, Consumer<String> displayResult) {
-        final Command command = addressBookParser.eagerEvaluateCommand(commandText);
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/LogicManager.java]: @CarbonGrid, I edited this part for the eager evalution

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -281,7 +281,7 @@ private void executeCommand(String commandText) {
      * Called to update AutoComplete when new commands.
      */
     public void updateCommandAutoComplete(String commandText) {
-        if (!commandText.isBlank() && !commandText.endsWith(" ")) {
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/ui/MainWindow.java]: This caused a bug where no command was eagerly evaluated if the command ends with space, such as 'patient 0  '

File: src/main/java/seedu/address/logic/LogicManager.java (hunk 1 above)
Comment (CarbonGrid) [reply src/main/java/seedu/address/logic/LogicManager.java]: This one is not a bug. It does not make sense to eager evaluate on for additional space.

Comment (CarbonGrid) [reply src/main/java/seedu/address/logic/LogicManager.java]: In a sense, we can specify, eager evaluation will not be run if there is no meaningful changes to the command in command box.

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (CarbonGrid) [reply src/main/java/seedu/address/ui/MainWindow.java]: This one is not a bug. It does not make sense to eager evaluate on for additional space.

In a sense, we should specify, eager evaluation will not be run if there is no meaningful changes to the command in command box.

**Pull request merged**